crates/blockifier/src/execution/contract_class.rs

```diff
@@ -263,10 +266,9 @@ pub struct ContractClassV1Inner {
 impl Encode for ContractClassV1Inner {
     fn encode(&self) -> Vec<u8> {
         let val = self.clone();
-        let entry_points_by_type = val
-            .entry_points_by_type
-            .into_iter()
-            .collect::<Vec<(EntryPointType, Vec<EntryPointV1>)>>();
+        let entry_point_btree = hashmap_to_btree(val.entry_points_by_type);
+        let entry_points_by_type =
+            entry_point_btree.into_iter().collect::<Vec<(EntryPointType, Vec<EntryPointV1>)>>();
```

Comment:
I don't think you need to do hashmap -> btree -> vec -> encode, you can do hashmap -> btree -> encode